Twilio/Page.php

```diff
@@ -40,7 +40,17 @@ public function __construct(Version $version, Response $response) {
 
     protected function processResponse(Response $response) {
         if ($response->getStatusCode() != 200 && !$this->isPagingEol($response->getContent())) {
-            throw new DeserializeException('Unable to fetch page', $response->getStatusCode());
+            $message = '[HTTP ' . $response->getStatusCode() . '] Unable to fetch page';
+            $code = $response->getStatusCode();
+
+            $content = $response->getContent();
+
+            if (is_array($content)) {
+                $message .= isset($content['message']) ? ': ' . $content['message'] : '';
+                $code = isset($content['code']) ? $content['code'] : $code;
+            }
+
+            throw new DeserializeException($message, $code);
```

Comment:
Ok, that makes sense, I'll make the change and call it out as a potentially backwards incompatibility (if you were explicitly catching `DeserializeException`)